octue/resources/analysis.py

```diff
@@ -12,6 +12,13 @@
 module_logger = logging.getLogger(__name__)
 
 
+HASH_FUNCTIONS = {
+    "configuration_values": Hashable.hash_non_class_object,
```

Comment:
Yet more reasons to make our Values objects something other than a vanilla dict.